Makefile

```diff
@@ -121,6 +121,75 @@ validate-cr:
 ## gen-crd-doc: Generates documentation for the Kiali CR and OSSMConsole CR configuration
 gen-crd-doc: .gen-crd-doc-kiali .gen-crd-doc-ossmconsole
 
+## sync-crds: Synchronizes all CRD files from the golden copies
+sync-crds:
+	@echo "Synchronizing Kiali CRD from golden copy: crd-docs/crd/kiali.io_kialis.yaml"
+
+	@if [ -d "../helm-charts/kiali-operator/crds" ]; then \
+		echo "  -> helm-charts/kiali-operator/crds/crds.yaml (with YAML document separators)"; \
+		echo "---" > ../helm-charts/kiali-operator/crds/crds.yaml; \
+		cat crd-docs/crd/kiali.io_kialis.yaml >> ../helm-charts/kiali-operator/crds/crds.yaml; \
+		echo "..." >> ../helm-charts/kiali-operator/crds/crds.yaml; \
+	else \
+		echo "  -> helm-charts/kiali-operator/crds/crds.yaml (SKIPPED - directory not found)"; \
+	fi
```

Comment:
Can you make `helm-charts` dir configurable? Mine is at `kiali-helm-charts` so this command fails for me. I wonder if it'd be easier long term to move the operator and helm-charts into `kiali/kiali`?

Reply:
If we don't want to make them configurable that's fine.

In practice, we always release the operator, server, and helm charts together right? The releases are all tied together and they share a lot of the same makefiles/hack scripts. Does it matter that the operator can also install older versions of the server?